components/python/pygobject-34/Makefile

```diff
@@ -0,0 +1,66 @@
+#
```

Comment:
Why did you choose pygobject-34 for packages that provide versions for python-3.7 and 3.9?
My recommendation is to use pygobject-legacy for the old python versions and pygobject (or pygobject-3) for the latest python versions.

Reply:
The "-34" suffix is derived from version 3.40.0 of pygobject.  It is not related to python-34 .

I assumed that it was not possible to rename existing component directories, without introducing complications.  Was I wrong?

I'm willing to do the two renames but, how do I do that, after having done the "git commit"?
I'd prefer to rename the old directory to pygobject-3-legacy and the new one to pygobject-3, instead.

Reply:
After you have changed your local git history you can simply use a force push (eg. "git push -f <branch name> origin") and Github will automatically update your PR.

Reply:
That was not my question.  I wanted to know what do I do locally before the push?  I've never done this before.  I want to to it right.

Obviously I have to be in the pygobject branch.  Should I just do "git mv" twice?  Or, should I undo the commit first?  After that, I'd do "git mv" twice, all the "git add/rm" commands, and the commit again.

Of course, I'd squash the commits before doing the forced push.